app/code/Bulbulatory/Recommendations/etc/sales.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<config>
+    <section name="quote">
+        <group name="totals">
+            <item name="discount" instance="Bulbulatory\Recommendations\Model\Total\Quote\RecommendationDiscount" sort_order="300"/>

```

Comment:
@TM18

> You should use a custom name for this total, as discount is already being used by discount codes. So now when I apply a discount code, I can't see it in the cart summary because it is overwritten by this.

I changed _discount_ to _custom_recommendation_discount_